### PR TITLE
DOCS-3147: Use resource-level logging for Python modules

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

**Review by npentrel — APPROVED** [review]
This looks good to me - was that the only occurrance of using the global logger in our docs?

File: docs/how-tos/create-module.md [hunk 1]
```diff
@@ -1119,7 +1119,24 @@ The example code shown above under [Write your new resource model definition](#w
...
+self.logger.debug("debug info")
+self.logger.info("info")
+self.logger.warn("warning info")
+self.logger.error("error info")
+self.logger.exception("error info", exc_info=True)
+self.logger.critical("critical info")
+```
+
+Resource-level logs are recommended instead of global logs for modular resources, because they make it easier to determine which component or service an error is coming from.
+Resource-level logs appear in the **ERROR LOGS** section of each resource's configuration card in the app.
```
Comment (JessamyT) [inline docs/how-tos/create-module.md:1135]:
```suggestion
Resource-level logs appear in the **ERROR LOGS** section of each resource's configuration card in the app.

{{% alert title="Note" color="note" %}}
In order to see resource-level debug logs when using your modular resource, you'll either need to run `viam-server` with the `-debug` option or [configure your machine or individual resource to display debug logs](/architecture/viam-server/#logging).
{{% /alert %}}
```

**Review by stuqdog — APPROVED** [review]
lgtm, thanks for updating these!

**Comment by JessamyT** [comment]
> This looks good to me - was that the only occurrance of using the global logger in our docs?

This and the sensor module page which also needs updating were the only places where I see it for Python

File: docs/how-tos/create-module.md [hunk 2]
```diff
@@ -1119,7 +1119,28 @@ The example code shown above under [Write your new resource model definition](#w
...
```
Comment (JessamyT) [inline docs/how-tos/create-module.md:1135]:
```suggestion
Resource-level error logs appear in the **ERROR LOGS** section of each resource's configuration card in the app.
```

**Review by npentrel — APPROVED** [review]

**Pull request merged**